tests/smoke-tests/package.json

```diff
@@ -18,8 +18,9 @@
     "node": "^20"
   },
   "dependencies": {
-    "@sap-ai-sdk/ai-core": "canary",
-    "@sap-ai-sdk/gen-ai-hub": "canary",
+    "@sap-ai-sdk/ai-api": "workspace:^",
```

Comment:
[q] I imagine this will need to be changed back to canary once we have the release for the renamed packages? Is there a ticket for it?

Reply:
True, I actually put a comment here but it was in pending 😅 I will create a separate PR to just revert these fields.